resources/definitions/3DI_D300_v2

```diff
@@ -0,0 +1,34 @@
+{
+    "version": 2,
+    "name": "3DI D300 v2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "V27Jain",
+        "manufacturer": "3Deometry Innovations",
+        "file_formats": "text/x-gcode",
+        "platform": "3DI_D300_v2_platform.STL",
```

Comment:
This platform is not aligned to the build volume. It needs to be centred.
![image](https://user-images.githubusercontent.com/2448634/76611810-60285600-651b-11ea-948d-ab2dfe8b52d1.png)